README.md

```diff
@@ -7,10 +7,7 @@ $ cd newproject
 # Use composer to prepare the core
 $ composer install
 
-# Use the concrete5 CLI tool to run the c5 install
-$ ./vendor/bin/concrete5 c5:install -i
+# Use the concrete5 CLI tool to run the c5 install from the public directory
+$ cd public
+$ ./concrete/bin/concrete5 c5:install -i
```

Comment:
I believe this needs to run in the public directory unfortunately. ./concrete/bin/concrete5 c5:install works okay